UI/forms/OBSBasicSettings.ui

```diff
@@ -38,7 +38,7 @@
        </property>
        <property name="maximumSize">
         <size>
-         <width>135</width>
+         <width>165</width>
```

Comment:
Another small thing, was it necessary to change the width?

Reply:
Yes, otherwise "Screenshots" would not fit in the list horizontally

Reply:
Alright